bin/katana/src/cli/node.rs

```diff
@@ -540,8 +614,9 @@ mod test {
     fn test_starknet_config_custom() {
         let args = NodeArgs::parse_from([
             "katana",
-            "--disable-fee",
-            "--disable-validate",
+            "--dev",
+            "--dev.no-fee",
+            "--dev.no-account-validation",
```

Comment:
does `--dev.no-fee` implies `--dev`, or `--dev` is first required to have those enabled? (the latter I guess)

Reply:
Ok just seen the code, any option with `--dev.*` triggers dev. Nice.